src/components/Body.js

```diff
@@ -6,6 +6,13 @@ const Body = props => {
     <div className="bg-light">
       <div className="body">
         <p className="tx-dark">{props.text}</p>
+        <p className="tx-dark">These links may be helpful:</p>
+        <a href="https://reactjs.org/docs/getting-started.html">
+          React Documention
+        </a>
+        <a href="https://webpack.js.org/concepts/">Webpack Documention</a>
```

Comment:
And then in the link:
```suggestion
        <a style={linkStyle} href="https://webpack.js.org/concepts/">Webpack Documention</a>
```